Cargo.toml

```diff
@@ -8,7 +8,7 @@ exclude = ["examples/*", "book/*"]
 keywords = ["game", "engine", "sdk", "amethyst"]
 categories = ["game-engines"]
 
-documentation = "https://www.amethyst.rs/doc/master/doc/amethyst"
+documentation = "https://www.amethyst-engine.org/doc/latest/doc/amethyst/"
```

Comment:
Are we moving to this domain? Has it been setup to build from `amethyst/website`?

Reply:
It was said that it was an eventuality, and with the sudden outage, I figured it would be prudent to prepare.

That, and I was testing TC's PR building.

And I'm not sure if it regularly builds/deploys from it, but it is currently usable.

Reply:
Can we update the links but remain on amethyst.rs? At least for now, though the links need to be updated to point to latest for release versions.

Reply:
I'm not sure. This PR is a preparation for something I know has been planned for, but is not something I can say we're ready for.

Reply:
@LucioFranco yes, we have to move to this domain, because we have no control over amethyst.rs, nor can we get it. Before merging, I'd like a checklist of everything we need to do to migrate to a different domain. This can wait until then.

Reply:
Right so what I was suggesting is that we use this to update the cargo and the links to actually point to latest instead of master. I know we will have to move but currently amethyst.rs works and as far as I can see amethyst-engine.org isn't fully setup yet.

Reply:
so I'd like to move this forward or I can open another that will make similar changes because we need to update the links anyways but we should still remain on the old domain until the new one is ready.

Reply:
Close this one for now, and do another one with the old links. Moving domains is going to require some coordination.